challenge_1/java/jdfurlan/src/ReverseAString.java

```diff
@@ -0,0 +1,18 @@
+
+import java.util.Scanner;
+
+public class ReverseAString {
+
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
```

Comment:
You could utilize a try-with-resources to guarantee closure of sc upon exiting the block.